### PR TITLE
fix variable error

### PR DESCRIPTION
**Pull request by xiaolou86**
1. fix variable error in `example/burrow_integration/permissions.py`
2. fix some typos

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -41,7 +41,7 @@ endif()
 
 ## FIXME revert this after change CI to GitHub actions.
 ## Temporal fix for 'bytecode stream version incompatible' between gcc-9 and gcc-10 and clang
```
Comment (appetrosyan) [inline CMakeLists.txt:43]:
```suggestion
## Temporary fix for 'bytecode stream version incompatible' between gcc-9 and gcc-10 and clang
```

**Review by appetrosyan — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (xiaolou86) [reply CMakeLists.txt]: @appetrosyan Thanks for your review.

**Comment by xiaolou86** [comment]
@appetrosyan Thanks for your review.

File: docs/source/build/index.rst [hunk 2]
```diff
@@ -95,7 +95,7 @@ Here are the steps:
 
   docker run -it hyperledger/iroha-builder:latest
 
-On this step you will start and run the image in a container in an interactive mode. You can use any available tags, default one would be ``latest``, and developmnet tag is ``develop``. Note that you might need to perform some actions with ``sudo`` rights. 
+On this step you will start and run the image in a container in an interactive mode. You can use any available tags, default one would be ``latest``, and developments tag is ``develop``. Note that you might need to perform some actions with ``sudo`` rights.
```
Comment (xiaolou86) [inline docs/source/build/index.rst:98]:
```suggestion
On this step you will start and run the image in a container in an interactive mode. You can use any available tags, default one would be ``latest``, and development tag is ``develop``. Note that you might need to perform some actions with ``sudo`` rights.
```

File: example/burrow_integration/permissions.py [hunk 3]
```diff
@@ -40,7 +40,7 @@ def create_contract():
     return hex_hash
 
 @integration_helpers.trace
-def grant_permission(adress, permission):
+def grant_permission(address, permission):
```
Comment (baziorek) [inline example/burrow_integration/permissions.py:43]: remember also about changing the name here:
`"CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params` (line 57)

File: example/burrow_integration/permissions.py [hunk 4]
```diff
@@ -66,7 +66,7 @@ def grant_permission(adress, permission):
     return hex_hash
 
 @integration_helpers.trace
-def revoke_permission(adress, permission):
+def revoke_permission(address, permission):
```
Comment (baziorek) [inline example/burrow_integration/permissions.py:69]: remember also about changing the name here:
` "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params` (line 83)

**Review by baziorek — CHANGES_REQUESTED** [review]
Thanks for correcting typos. Just two small changes to fix after renaming

File: example/burrow_integration/permissions.py (hunk 3 above)
Comment (xiaolou86) [reply example/burrow_integration/permissions.py:43]: It was already named `address`. so I think it maybe contains a variable issue here.

File: example/burrow_integration/permissions.py (hunk 4 above)
Comment (xiaolou86) [reply example/burrow_integration/permissions.py:69]: It was already named `address`.

**Comment by xiaolou86** [comment]
@baziorek thank you very much for your review.

**Review by baziorek — APPROVED** [review]
LGTM

File: example/burrow_integration/permissions.py (hunk 4 above)
Comment (baziorek) [reply example/burrow_integration/permissions.py:69]: You are right. I don't know why I didn't notice:D.

File: example/burrow_integration/permissions.py (hunk 3 above)
Comment (baziorek) [reply example/burrow_integration/permissions.py:43]: You are right. I don't know why I didn't notice:D.

**Comment by xiaolou86** [comment]
@appetrosyan @baziorek thanks for your reviews. Could you help to merge?

**Comment by baziorek** [comment]
> @appetrosyan @baziorek thanks for your reviews. Could you help to merge?

It was merged already by @6r1d . Thanks @xiaolou86 for Your work!

**Pull request merged**